pyproject.toml

```diff
@@ -30,6 +30,7 @@ dependencies = [
     "pydantic~=1.10.0",
```

Comment:
Why are we pinning the minor versions in these packages?

I.E. why "~=<MAJOR>.<MINOR>.<SUBMINOR>" instead of ">=<MAJOR>.<MINOR>.<SUBMINOR>"?